draft-packages/likert-scale-legacy/KaizenDraft/LikertScaleLegacy/LikertScaleLegacy.module.scss

```diff
@@ -249,6 +265,47 @@ $fifth: $color-red-500;
   border-color: rgba($color-white-rgb, 0.65);
 }
 
+.check {
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin: -10px 0 0 -10px;
+  color: black;
+
+  .likertItem1 & {
+    margin-left: -9px;
+
+    [dir="rtl"] & {
+      margin-left: -11px;
+    }
+  }
+
+  .likertItem5 & {
+    margin-left: -12px;
+
+    [dir="rtl"] & {
+      margin-left: -9px;
+    }
+  }
+}
+
+:where(.likertItem4, .likertItem5) .check {
+  color: white;
+}
+
+.hovered :where(.likertItem4, .likertItem5) .check {
+  color: black;
+}
+
+:is(.likertItem4.suggested, .likertItem5.suggested) .check {
+  color: white;
+}
+
+.hovered.reversed .itemContainer > :not(.suggested.likertItem) .check {
+  color: white;
+}
```

Comment:
Hmm might be cleaner to use JS to conditionally add a `checkmark-white` classname when given X condition.

```
likertId < 6 && .checkmark-white
```

Then you can contain your styles:

```
.checkmark-white .check {}
.checkmark-white:hover .check {}
```

Reply:
Or go for a `likertItemDark` and `likertItemLight`, applied on the appropriate items. This could also help you reduce the classes you need to reference + scales better if more items were to be added or the colour needs to change, you'll have a better indicator of where to apply the right changes, rather than just numbers

Reply:
<img width="240" alt="image" src="https://user-images.githubusercontent.com/763385/230280106-643415ae-6862-4879-918c-2caaf85c1e0b.png">
<img width="271" alt="image" src="https://user-images.githubusercontent.com/763385/230280197-5e79b65c-ec8b-4528-9921-6870c045e893.png">

Some background, I don't think it's obvious what's going on here:

The check on the last two item (4 + 5) is white, because of their darker background colors. But when you hover on the scale and they potentially have a transparent background, they need to be dark.

But when `reversed` those last two stay white (because the bg is dark) and also the other ones that are usually dark need to be white 🤯

The JS logic for that is also not going to be trivial, but at least testable - which is my biggest gripe with a pure CSS solution.

I'll give it a quick go, but not sure I can finish it today.

Reply:
Maybe not worth the stab then, at least we have chromatic.

Reply:
I was just giving it a go, but the function would need:
- current item's value
- value of hovered item (or null)
- value of selected item (or null)
- and reversed state
From there it would determine the check mark's color.

Just the test cases will take a while to write, unless I can come up with something cleverer than in the past 10 min ;)